app/controllers/planning_application/permitted_development_rights_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class PlanningApplication
-  class PermittedDevelopmentRightsController < AuthenticationController
+  class PermittedDevelopmentRightsController < AuthenticationController # rubocop:disable Metrics/ClassLength
```

Comment:
I wonder if we should just remove this check (and related ones) — it seems like we're just going to disable it whenever it crops up. (And if I'm honest that seems like the most reasonable response anyway.)

Reply:
Yeah agreed, will add a commit to remove it